### PR TITLE
Readme typo fixes

### PR DESCRIPTION
**Pull request by kolenic-martin**
This fixes two minor issues in the recent README update.

### CODE REVIEW

File: packages/airnode-deployer/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ cp config/secrets.example.env config/secrets.env
 # Edit both `config.json` and `secrets.env` to reflect your configuration
 ```
 
-After, you have the necessary deployment files ready, there are two options how to run the CLI:
+After you prepare the necessary deployment files, there are two options how to run the CLI:
```
Comment (dcroote) [inline packages/airnode-deployer/README.md:37]:
```suggestion
After you prepare the necessary deployment files, there are two options for running the CLI:
```

**Comment by dcroote** [comment]
Minor suggestion for the second half of the sentence

**Comment by kolenic-martin** [comment]
> Minor suggestion for the second half of the sentence

Thanks @dcroote, this sounds a lot better.

**Comment by amarthadan** [comment]
@kolenic-martin @dcroote Are we waiting for something here?

**Comment by dcroote** [comment]
Nope I think it's all set

**Pull request merged**